mne/viz/evoked.py

```diff
@@ -1355,6 +1352,32 @@ def _ci(arr, ci):
                        for mean_, sigma_ in zip(mean, sigma)]).T
 
 
+def _bootstrap_ci(arr, ci=.95, n_bootstraps=2000, statfun='mean'):
+    """Get confidence intervals from non-parametric bootstrap."""
+    if statfun == "mean":
+        statfun = lambda x: x.mean(axis=0)  # noqa
+    elif statfun == 'median':
+        statfun = lambda x: np.median(x, axis=0)  # noqa
+    elif not callable(statfun):
+        raise ValueError("statfun must be 'mean', 'median' or callable.")
+    ntrials, ntimes = arr.shape
+    indices = np.arange(ntrials, dtype=int)  # BCA would be cool to have too
+    boot_indices = np.random.choice(indices, replace=True,
+                                    size=(ntrials, len(indices)))
+    stat = np.array([statfun(arr[inds]) for inds in boot_indices])
+    ci = (((1 - ci) / 2) * 100, ((1 - ((1 - ci) / 2))) * 100)
+    ci_low, ci_up = np.percentile(stat, ci, axis=0)
+    return np.array([ci_low, ci_up])
```

Comment:
We should probably move these to `mne.stats` somewhere. They will be used by other things (including the Welch PR @mmagnuski mentioned)

Reply:
I can sort it out in welch PR or a subsequent one.

Reply:
Okay let's do it in the Welch PR, then, this one seems closer